### PR TITLE
Update Main.cpp

### PR DESCRIPTION
**Pull request by xuehyc**
**Changes proposed:**

-  
-  
-  

**Target branch(es):** 3.3.5/master

- [ ] 3.3.5
- [ ] master

**Issues addressed:**

Closes #  (insert issue tracker number)

**Tests performed:**

(Does it build, tested in-game, etc.)

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

File: src/server/worldserver/Main.cpp [hunk 1]
```diff
@@ -72,6 +72,7 @@ namespace fs = boost::filesystem;
 #ifdef _WIN32
 #include "ServiceWin32.h"
 char serviceName[] = "worldserver";
+//服务器名称设置
```
Comment (acidmanifesto) [inline src/server/worldserver/Main.cpp:75]: What does server name in chinese have to do with anything?

**Comment by xuehyc** [comment]
在 2021-09-27 22:30:18，"acidmanifesto" ***@***.***> 写道：

@acidmanifesto commented on this pull request.

In src/server/worldserver/Main.cpp:

> @@ -72,6 +72,7 @@ namespace fs = boost::filesystem;

 #ifdef _WIN32

 #include "ServiceWin32.h"

 char serviceName[] = "worldserver";

+//服务器名称设置



What does server name in chinese have to do with anything?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by MaxtorCoder** [comment]
I am so confused.

**Comment by xuehyc** [comment]
In fact,many chinese use this program.But they may have difficulties in reading this.I think by this way,we can make it easier for them to read.



At 2021-09-28 21:04:37, "MaxtorCoder" ***@***.***> wrote:

I am so confused.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Faq** [comment]
Or they could learn few words in eng :)

**Pull request closed without merging**